topi/python/topi/x86/conv2d.py

```diff
@@ -523,3 +523,15 @@ def traverse(op):
 
     traverse(outs[0].op)
     return s
+
+@autotvm.register_topi_compute(nn.group_conv2d_nchw, 'cpu', 'direct')
```

Comment:
we don't need to register this as autotvm template since you are using depthwise tag

Reply:
Yes, I agree. I've just removed it.